src/Repository/CommissionableAssetRepository.php

```diff
@@ -85,6 +85,10 @@ public function findByFilters(array $formData, bool $onlyIds = false): array
             $qb = $this->addAvailabilityBetween($qb, $formData['availableFrom'], $formData['availableTo'], CommissionableAssetAvailability::class, 'asset');
         }
 
+        if (($formData['withoutAvailabilities'] ?? false) === false && empty($formData['availableFrom']) && empty($formData['availableTo'])) {
```

Comment:
Please use Yoda condition (cf. [Symfony Coding Standards](https://symfony.com/doc/current/contributing/code/standards.html#structure)):
```suggestion
        if (false === ($formData['withoutAvailabilities'] ?? false) && empty($formData['availableFrom']) && empty($formData['availableTo'])) {
```